samples/snippets/src/test/java/com/example/spanner/EncryptionKeyIT.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Integration tests for: - {@link CreateDatabaseWithEncryptionKey} - {@link
+ * CreateBackupWithEncryptionKey} - {@link RestoreBackupWithEncryptionKey}
+ */
+@RunWith(JUnit4.class)
+public class EncryptionKeyIT {
+
+  private static String projectId;
+  private static final String instanceId = System.getProperty("spanner.test.instance");
+  private static DatabaseAdminClient databaseAdminClient;
+  private static List<String> databasesToDrop;
+  private static List<String> backupsToDrop;
+  private static Spanner spanner;
+  private static String key;
+
+  @BeforeClass
+  public static void setUp() {
+    final SpannerOptions options = SpannerOptions
+        .newBuilder()
+        .setAutoThrottleAdministrativeRequests()
+        .build();
+    projectId = options.getProjectId();
+    spanner = options.getService();
+    databaseAdminClient = spanner.getDatabaseAdminClient();
+    databasesToDrop = new ArrayList<>();
```

Comment:
nit: This can be made `final` and assigned on declaration.

Reply:
Done.